modules/renter/hostdb/hostdb.go

```diff
@@ -17,6 +17,7 @@ import (
 	"github.com/NebulousLabs/Sia/persist"
 	siasync "github.com/NebulousLabs/Sia/sync"
 	"github.com/NebulousLabs/Sia/types"
+	"math"
```

Comment:
needs to be added to the first batch of imports. Sorting is in 3 batches: standard library imports, followed by internal imports (github.com/NebulousLabs/Sia/*) followed by external imports (github.com/NebulousLabs/fastrand for example)